xml/templates/message_templates/participant_cancelled_subject.tpl

```diff
@@ -1 +1 @@
-{ts 1=$event.event_title}Event Registration Cancelled for %1{/ts} - {contact.display_name}
+{ts 1={event.title}}Event Registration Cancelled for %1{/ts} - {contact.display_name}
```

Comment:
I think the test fails are from here, but that's spooky then that it ever worked without quotes?

Reply:
Oh, because it was a smarty var, so it knows. So now it needs quotes.